goben/client.go

```diff
@@ -6,12 +6,14 @@ import (
 	"encoding/gob"
 	"fmt"
 	"io"
+	"os"
 	"log"
 	"math/rand"
 	"net"
 	"runtime"
 	"sync"
 	"time"
+	"unicode"
```

Comment:
What do you need the Unicode package for?

Reply:
done